examples/style_sheets/style_sheets_reference.py

```diff
@@ -98,46 +99,45 @@ def plot_histograms(ax, prng, nb_samples=10000):
     return ax
 
 
-def plot_figure(style_label=""):
+def plot_figure(axs, style_label=""):
     """Setup and plot the demonstration figure with a given style."""
     # Use a dedicated RandomState instance to draw the same "random" values
     # across the different figures.
     prng = np.random.RandomState(96917002)
-
-    # Tweak the figure size to be better suited for a row of numerous plots:
-    # double the width and halve the height. NB: use relative changes because
-    # some styles may have a figure size different from the default one.
-    (fig_width, fig_height) = plt.rcParams['figure.figsize']
-    fig_size = [fig_width * 2, fig_height / 2]
-
-    fig, axs = plt.subplots(ncols=6, nrows=1, num=style_label,
-                            figsize=fig_size, squeeze=True)
-    axs[0].set_ylabel(style_label)
+    axs[0].figure.set_facecolor(plt.rcParams['figure.facecolor'])
```

Comment:
This is a workaround because all subfigures are created with one style. This will not be needed for standard use cases where one has one figure only. I think this workaround needs a comment.

Are there more figure-based parameters that would need to be adapted?